nservicebus/sqlserver/multiple-databases.md

```diff
@@ -1,21 +1,66 @@
 ---
-title: Multi-database support
-summary: How to configure SQL Server transport to use multiple instances of the database and route messages between them.
+title: SQL Server transport setup - default, multi-schema, multi-catalog and multi-instance
```

Comment:
should we change the name of md file as well and keep redirects for old `multiple-database.md`